libfabric.spec

```diff
@@ -1,15 +1,14 @@
 %define suse_libname libfabric1
 %global major 1
 %global minor 15
-%global bugrelease 0
-%global prerelease rc3
```

Comment:
I'm assuming you want this re-reviewed.  Please don't forget to click the re-review button when you are done making changes and ready for another review.  Otherwise your PR does not show up in people's to-review list.

Reply:
Thank you very much! I was waiting for the CI test PRs.

Reply:
Ahhh.  Fair enough.  Just didn't want to see us in a dead-lock, you waiting for a re-review and me waiting for a re-review-request.